errors/sendresultsmapper.go

```diff
@@ -0,0 +1,277 @@
+package errors
+
+// SendResultsMapper is a hashmap of all possible results when sending an sms
+var SendResultsMapper = map[string]map[string]string{
+	"200": {
+		"name":        "Success code",
+		"description": "Sent SMS",
+	},
+	"401": {
+		"name":        "Authentication error",
+		"description": "Authorization Required",
+	},
+	"414": {
+		"name":        "Failed to send SMS",
+		"description": "URL is longer than 8,190 byte",
+	},
+	"550": {
+		"name":        "Failed to send SMS",
+		"description": "Failed to send SMS",
+	},
+	"555": {
+		"name":        "Block IP address",
+		"description": "Your IP address has been blocked",
+	},
+	"560": {
+		"name":        "Mobile number",
+		"description": "Mobile number is invalid",
+	},
+	"570": {
```

Comment:
in the name can we remove spaces and do more code to avoid any "space" character issue and case insensitive? For example. `sms_send_failure`, `sms_size_exhausted`, `sms_text_id_invalid` etc?